bin/metrics-redis-graphite.rb

```diff
@@ -97,6 +97,12 @@ class Redis2Graphite < Sensu::Plugin::Metric::CLI::Graphite
          long: '--skipkeys KEYS',
          default: nil
 
+  option :conn_failure_status,
+         long: '--conn-failure-status EXIT_STATUS',
+         description: 'Exit status for Redis connection failures',
```

Comment:
This could be improved a bit to say "Returns the following exit status for Redis connection failures" which seems a bit more descriptive and does not me to jump to the code to understand what you mean.